install/infra/modules/eks/kubernetes.tf

```diff
@@ -102,6 +110,7 @@ module "eks" {
       enable_bootstrap_user_data = true
       instance_types             = [var.service_machine_type]
       name                       = "service-${var.cluster_name}"
+      iam_role_name              = format("%s-%s", substr("${var.cluster_name}-svc-ng", 0, 58), random_string.ng_role_suffix.result)
```

Comment:
Am I correct in reading that this it to make a string a max of 62 characters long, always ending `-svc-ng-<4-random-chars>`?

Reply:
that is correct. I believe that is the max char limit.